packages/react-app/src/components/quest.tsx

```diff
@@ -554,6 +561,20 @@ export default function Quest({
       <WrapperStyled>
         {!isEdit && (
           <>
+            <FirstRowStyled className="pb-0">
+              <HeaderRowStyled>
+                <StateTag state={questData?.state ?? ''} />
+              </HeaderRowStyled>
+              <HeaderRowStyled>
+                <AmountFieldInput
+                  id={`balance-${questData?.bounty?.parsedAmount}`}
+                  key={`balance-${questData?.bounty?.parsedAmount}`}
+                  amountLabel={questData?.bounty?.parsedAmount.toString()}
+                  compact
+                  value={questData?.bounty}
```

Comment:
Je viens de rajouter des prop pour mieux controller le behavior de AmountFieldInput so fo add sa astheure

```suggestion
                  value={questData?.bounty}
                  showUsd
                  onlyTag
```